.github/workflows/use-prod-sha-for-demo.yml

```diff
@@ -0,0 +1,40 @@
+name: Sync Demo Hash with Production
```

Comment:
```suggestion
name: Create PR to Sync Demo Hash with Production
```

Reply:
(IMO it's confusing if the action suggests it has a direct action and it doesn't)